### ISSUE TITLE
get_multi() throws "TypeError: unhashable type: 'Entity' "

### ISSUE THREAD

**Issue by JorinTielen** [Bogdanp/anom-py#16]
When using `get_multi()` with a lot of keys (probably around a 100), `get_multi()` will fail with this error:

```
Traceback (most recent call last):
  File "/env/lib/python3.7/site-packages/webapp2.py", line 604, in dispatch
    return method(*args, **kwargs)
...
  File "/srv/my_handler.py", line 231, in handle
    entities = anom.get_multi(keys)
  File "/env/lib/python3.7/site-packages/anom/model.py", line 716, in get_multi
    entities_data, entities = adapter.get_multi(keys), []
  File "/env/lib/python3.7/site-packages/anom/adapters/datastore_adapter.py", line 139, in get_multi
    request_keys.remove(key)
TypeError: unhashable type: 'Entity'
```
It seems an `Entity` somehow got in the list of `missing`.

I have verified it works fine if i just loop through the keys and `.get()` them individually (though it is damn slow).  I have also verified that all the keys passed are of type `anom.Key`.

**Comment by Bogdanp** [comment]
https://github.com/Bogdanp/anom-py/blob/master/anom/adapters/datastore_adapter.py#L138-L139 seems like this might be a bug in the gcloud libs. We'll be able to verify if it's fixed once #15 is done.

**Comment by Bogdanp** [comment]
Please give this another try with version 0.9 and reopen the issue if it's still happening. Thanks!
